docs/snippets/angular/msw-addon-configure-handlers-graphql.ts.mdx

```diff
@@ -58,28 +58,38 @@ type Story = StoryObj<DocumentScreen>;
 
 export const MockedSuccess: Story = {
   parameters: {
-    msw: [
-      graphql.query('AllInfoQuery', (req, res, ctx) => {
-        return res(ctx.data(TestData));
-      }),
-    ],
+    msw: {
+      handlers: [
+        graphql.query('AllInfoQuery', () => {
+          return new HttpResponse.json({
+            data: {
+              allFilms: {
+                films,
+              },
+            }
+          });
+        }),
+      ],
+    },
   },
 };
 
 export const MockedError: Story = {
   parameters: {
-    msw: [
-      graphql.query('AllInfoQuery', (req, res, ctx) => {
-        return res(
-          ctx.delay(800),
-          ctx.errors([
-            {
-              message: 'Access denied',
-            },
-          ]),
-        );
-      }),
-    ],
+    msw: {
+      handlers: [
+        graphql.query('AllInfoQuery', async () => {
+          delay(800);
```

Comment:
this should be awaited

Also don't forget to add `delay` to the msw imports at line 11